polaris-service/src/test/java/org/apache/polaris/service/PolarisApplicationIntegrationTest.java

```diff
@@ -179,6 +198,14 @@ public static void deletePrincipalRole() {
         .close();
   }
 
+  private static void assertZeroErrorsInApplicationLog() {
```

Comment:
Do we need to check archived log file as well?

Reply:
We don't.  I removed the Archived Log config.